test/test_clients_api.js

```diff
@@ -34,130 +35,134 @@ describe('Clients API', () => {
           });
   });
 
+  beforeEach(() => {
+    return db().then(models => {
+      models.Clients.destroy({ where: {} });
+    });
+  });
```

Comment:
You need to return `destroy` because otherwise the beforeEach finishes before the destroy is done.

using co-mocha you can do it like this:
```javascript
beforeEach(function*() {
  const { Clients } = yield db();
  yield Clients.destroy({ where: {} });
});
```